src/api/tests/test_available.py

```diff
@@ -101,15 +101,16 @@ def test_available_domain_dotgov(self):
         self.assertTrue(json.loads(response.content)["available"])
 
     def test_error_handling(self):
-        """Calling with bad strings raises an error."""
+        """Calling with bad strings returns error (error verifying) or unavailable (invalid domain)."""
         bad_string = "blah!;"
         request = self.factory.get(API_BASE_PATH + bad_string)
         request.user = self.user
         response = available(request, domain=bad_string)
         self.assertFalse(json.loads(response.content)["available"])
-        # domain set to raise error returns false for availability
-        error_domain_available = available(request, "errordomain.gov")
-        self.assertFalse(json.loads(error_domain_available.content)["available"])
+        # domain set to raise error returns false for availability and error message
+        error_domain_response = available(request, domain="errordomain.gov")
```

Comment:
This should be it's own test since it's a different output we are checking

Reply:
Updated to split bad string inputs and error handling into separate test cases